styles/desktop-version.css

```diff
@@ -423,6 +423,79 @@ html {
     display: none;
   }
 
+  /* modal project details */
+  .modal-project {
+    background-color: #1a2236;
+  }
+
+  .modal-desktop-container {
+    background: #fff;
+    width: 85%;
+    height: auto;
+    margin: 7.2% auto;
+  }
+
+  .modal-project-name {
+    font-size: 40px;
+    text-align: left;
+    margin-top: -8%;
+    margin-bottom: 5%;
+    padding-left: 5%;
+  }
+
+  .modal-stack-list {
+    justify-content: flex-start;
+    padding-left: 5%;
+    gap: 3%;
+  }
+
+  .modal-project-details-text {
+    width: 70%;
+    margin: 0 auto;
+  }
+
+  .modal-project-details-text-h3 {
+    font-size: 15px;
+  }
```

Comment:
For the popup modal for the desktop view, you need to further adjust your CSS code to make it like the Figma design. 
I notice that it looks perfect for the mobile view except and that you have not used media query to adjust it on the desktop view. Kindly fix this.

Figma design | Your design
---- | ----
![image](https://user-images.githubusercontent.com/84156714/223734397-bdd8f224-c12d-401e-8ba4-1d96f639fd5f.png) | ![image](https://user-images.githubusercontent.com/84156714/223735183-b3dded97-3712-4542-bf42-25724b382759.png)

Reply:
modal popup for desktop version is finished and still on **modal-project-details** branch, in main branch is not existed yet until we **merge** it from.